plugins/janus_streaming.c

```diff
@@ -3264,6 +3274,7 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 					(char *)json_string_value(url),
 					username ? (char *)json_string_value(username) : NULL,
 					password ? (char *)json_string_value(password) : NULL,
+					quirk ? TRUE : FALSE,
```

Comment:
Same mistake as above. If you check how we deal with boolean properties in JSON requests, we do something like this:

    gboolean dovideo = video ? json_is_true(video) : FALSE;

so you should do something similar with your new feature, and pass the boolean to `janus_streaming_create_rtsp_source`.